readthedocs/builds/models.py

```diff
@@ -184,10 +184,12 @@ def get_downloads(self, pretty=False):
         return data
 
     def get_conf_py_path(self):
-        conf_py_path = self.project.conf_file(self.slug)
-        conf_py_path = conf_py_path.replace(
-            self.project.checkout_path(self.slug), '')
-        return conf_py_path.replace('conf.py', '')
+        conf_py_path = self.project.conf_dir(self.slug)
+        if not conf_py_path.endswith('/'):
+            conf_py_path += '/'
```

Comment:
This should be `os.path.sep` or rely on `os.path.join`. Also, if you use `os.path.relpath` below, you shouldn't need this at all as it handles normalizing paths a little bit